packages/cli-platform-ios/src/commands/buildIOS/index.ts

```diff
@@ -44,6 +45,21 @@ function buildIOS(_: Array<string>, ctx: Config, args: FlagsT) {
 
   const {xcodeProject, sourceDir} = ctx.project.ios;
 
+  if (args.explicit) {
+    const {scheme, mode} = await runExplicitMode({
+      scheme: args.scheme,
+      mode: args.mode,
+    });
+
+    if (scheme) {
+      args.scheme = scheme;
+    }
+
+    if (mode) {
+      args.mode = mode;
```

Comment:
Let's avoid mutating args. Instead create variables and use that for the rest of the code below